packages/amplication-server/src/core/workspace/workspace.resolver.ts

```diff
@@ -93,6 +93,10 @@ export class WorkspaceResolver {
       currentUser
     );
 
+    await this.workspaceService.dataMigrateWorkspaceResourcesCustomDtos(
+      currentUser.workspace?.id
+    );
```

Comment:
As we are passing a potentially undefined parameter, we should always check it the method call or first thing inside the method logic (param validation)